src/Components/MyLists/MyLists.tsx

```diff
@@ -13,12 +13,12 @@ const MyLists: React.FC<Props> = (props: Props) => {
   
   // maybe make one dynamic function to create cards
 
-  const gottaGoCards = {};
+  // const gottaGoCards = {};
     // map over userLists.gottaGo
     // for each rest. in list, create a Card
     // will be an array of card components
 
-  const lovedItCards = {};
+  // const lovedItCards = {};
     // map over userLists.lovedIt
     // for each rest. in list, create a Card
     // will be an array of card components
```

Comment:
Great work on routing, let's remember to add a switch to the MyLists component when we get all the lists added